sdk/lib/rtl/actctx.c

```diff
@@ -1914,6 +1946,9 @@ static void parse_cominterface_proxy_stub_elem( xmlbuf_t *xmlbuf, struct dll_red
         else if (xml_attr_cmp(&attr, threadingmodelW))
         {
         }
```

Comment:
Conditionals with null functions look suspicious for the above and below. Adding a comment might be helpful.
It could be "/* Nothing to be done here. */" or "/* What should be done here? */".
This could tell us if there is really nothing to be done, or that we might have an unimplemented code path.

Reply:
Yep, if this is actually an unimplemented code path, then an `FIXME()` debug print macro with some text can be added instead. But in case it's not investigated yet, then it makes sense to add a `/* FIXME: ... */` prefix in the comment. This will indicate this code path needs to be investigated still.